plutus-benchmark/nofib/exe/Main.hs

```diff
@@ -254,12 +247,13 @@ main = do
           Prime input             -> print $ Prime.runFixedPrimalityTest input
           Primetest n             -> if n<0 then P.error "Positive number expected"
                                      else print $ Prime.runPrimalityTest n
-    DumpPLC pa -> mapM_ putStrLn $ unindent . PLC.prettyPlcClassicDebug $ getWrappedProgram pa
```

Comment:
This was getting back a deBruijnified program and just writing it out without converting to proper names.